stories/12-Accordion.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {Fragment} from "react";
+import {storiesOf} from "@storybook/react";
+import Accordion from "../src/accordion/Accordion";
+
+storiesOf("Accordion", module).add("Accordion", () => (
+  <Fragment>
+    <p>
+      {
+        'Accordion with defaultExpanded={["1"]} allowZeroExpanded={false} allowMultipleExpanded={false}'
```

Comment:
Let's use double quotes `"` for string expressions.

```suggestion
        "Accordion with defaultExpanded={["1"]} allowZeroExpanded={false} allowMultipleExpanded={false}"
```